metrics.go

```diff
@@ -26,16 +26,9 @@ func (t *timer) Emit() {
 		return
 	}
 
-	go func(key string, start, end time.Time) {
-		select {
-		case statQueue <- NewTimer(
-			fmt.Sprintf("%s.%s", metricPrefix, key),
-			start,
-			end,
-		):
-		default:
-		}
-	}(t.key, t.start, time.Now())
+	statQueue <- NewTimer(fmt.Sprintf("%s.%s", metricPrefix, t.key),
```

Comment:
I would be hestitant to remove publishing metrics to the queue without a go routine, which yes could mean metrics are lost I suppose.  BUT if for some reason statsite crashes, or is slow accepting metrics and the statsQueue fills up, these calls will block until there is room in the queue effecting halting the app.

Statsite failures should not stop the application from running.